scripts/spectrum-vars.js

```diff
@@ -20,49 +20,67 @@ const license = fs.readFileSync(
 );
 
-const processCSS = (srcPath, dstPath, identifier) => {
-    fs.readFile(srcPath, 'utf8', function(err, data) {
-        if (err) {
-            return console.log(err);
-        }
-
-        /* lit-html is a JS litteral, so `\` escapes by default.
-         * for there to be unicode characters, the escape must
-         * escape itself...
-         */
-        let result = data.replace(/\\/g, '\\\\');
-
-        // possible selectors to replace
-        const selector1 = `.spectrum--${identifier}`;
-        const selector2 = '.spectrum';
-
-        // new selector values
-        const shadowSelector = ':root,\n:host';
-
-        if (data.indexOf(selector1) >= 0) {
-            result = result.replace(selector1, shadowSelector);
-        } else if (data.indexOf(selector2) >= 0) {
-            result = result.replace(selector2, shadowSelector);
-            result = result.replace(
-                `.spectrum--medium,
+const processCSSData = (data, identifier) => {
+    /* lit-html is a JS litteral, so `\` escapes by default.
+     * for there to be unicode characters, the escape must
+     * escape itself...
+     */
+    let result = data.replace(/\\/g, '\\\\');
+
+    // possible selectors to replace
+    const selector1 = `.spectrum--${identifier}`;
+    const selector2 = '.spectrum';
+
+    // new selector values
+    const shadowSelector = ':root,\n:host';
+
+    if (data.indexOf(selector1) >= 0) {
+        result = result.replace(selector1, shadowSelector);
+    } else if (data.indexOf(selector2) >= 0) {
+        result = result.replace(selector2, shadowSelector);
+        result = result.replace(
+            `.spectrum--medium,
 .spectrum--large`,
-                shadowSelector
-            );
-            result = result.replace(
-                `.spectrum--darkest,
+            shadowSelector
+        );
+        result = result.replace(
+            `.spectrum--darkest,
 .spectrum--dark,
 .spectrum--light,
 .spectrum--lightest`,
-                shadowSelector
-            );
+            shadowSelector
+        );
+    }
+
+    return result;
+};
+
+const processCSS = (srcPath, dstPath, identifier) => {
+    fs.readFile(srcPath, 'utf8', function(err, data) {
+        if (err) {
+            return console.log(err);
         }
 
+        let result = processCSSData(data, identifier);
         result = `${license}\n/* stylelint-disable */\n${result}\n/* stylelint-enable */`;
 
         fs.writeFile(dstPath, result, 'utf8');
     });
 };
 
+// For fonts.css we need to combine 2 source files into 1
+const processMultiSourceCSS = (srcPaths, dstPath, identifier) => {
+    let result = '';
+
+    for (const srcPath of srcPaths) {
+        let data = fs.readFileSync(srcPath, 'utf8');
+        result = `${result}\n${processCSSData(data, identifier)}`;
+    }
+
+    result = `${license}\n/* stylelint-disable */\n${result}\n/* stylelint-enable */`;
+    fs.writeFile(dstPath, result, 'utf8');
```

Comment:
I hope we don't have to edit this again, but can we centralize these two lines (same as 65 and 67) just in case we need to alter the structure of the result in the future?